src/DynamoCore/Graph/ConnectorPinModel.cs

```diff
@@ -7,12 +7,17 @@ namespace Dynamo.Graph
 {
     public class ConnectorPinModel : ModelBase
     {
-        private double height = 30;
-        private double width = 30;
+        private const double height = 30;
+        private const double width = 30;
         public Guid ConnectorId { get; set; }
+        /// <summary>
+        /// Required when needing to pull a reliable value for objects of this
+        /// type when there is no instances of this object type.
+        /// </summary>
+        public const double StaticWidth = width;
 
         public override double Height { get => height; }
-        public override double Width { get => width; }
+        public override double Width { get => StaticWidth; }
```

Comment:
What's the purpose of this change? To me seems no need to change?

Reply:
Honestly just setting a `const` there so that I can access a value corresponding the the class without there needing to be an instance of the class. Perhaps it is best if I make it a `static` property instead? On line 20 just making use of that variable as opposed to pointing to yet another thing, but agreed I can reveft line 15 to what it was previously.